src/main.cpp

```diff
@@ -1,21 +1,24 @@
 #include <Arduino.h>
```

Comment:
Please migrate main code here to its own test file that we can invoke individually

Reply:
Investigate what testing suite or general testing support is offered by PlatformIO